hal/lib/athena/Relay.cpp

```diff
@@ -72,9 +70,10 @@ void freeRelayPort(HalRelayHandle relay_port_handle) {
 }
 
 bool checkRelayChannel(uint8_t pin) {
-  return pin < kRelayHeaders;  // roboRIO only has 4 headers, and the FPGA has
-                               // seperate functions for forward and reverse,
-                               // instead of seperate pin IDs
+  return pin <
+         kNumRelayHeaders;  // roboRIO only has 4 headers, and the FPGA has
+                            // seperate functions for forward and reverse,
+                            // instead of seperate pin IDs
```

Comment:
Ugh... format.py messes this up... I wonder if there's a way to change its behavior with comments.